docs/attributes-registry/source.md

```diff
@@ -1,21 +1,16 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Source
 --->
 
-# Source Attributes
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
 
-These attributes may be used to describe the sender of a network exchange/packet. These should be used
-when there is no client/server relationship between the two sides, or when that relationship is unknown.
-This covers low-level network interactions (e.g. packet tracing) where you don't know if
-there was a connection or which side initiated it.
-This also covers unidirectional UDP flows and peer-to-peer communication where the
-"user-facing" surface of the protocol / API does not expose a clear notion of client and server.
```

Comment:
Also gone, maybe it can be brought back by adding it to the attribute group notes?

Reply:
We aren't rendering attribute group notes, but I can add that if you want.

Reply:
Group notes are now included in template.